app/src/main/java/io/intrepid/contest/screens/sendinvitations/selectcontacts/SelectContactsPresenter.java

```diff
@@ -1,12 +1,17 @@
 package io.intrepid.contest.screens.sendinvitations.selectcontacts;
 
 import android.support.annotation.NonNull;
+import android.support.v7.widget.SearchView;
+
+import java.util.ArrayList;
+import java.util.List;
 
 import io.intrepid.contest.base.BasePresenter;
 import io.intrepid.contest.base.PresenterConfiguration;
+import io.intrepid.contest.models.Contact;
 
 public class SelectContactsPresenter extends BasePresenter<SelectContactsContract.View>
-        implements SelectContactsContract.Presenter {
+        implements SelectContactsContract.Presenter, SearchView.OnQueryTextListener {
```

Comment:
As mentioned in the other PR, have the Contract class extend the listeners so you don't have to cast the presenter when setting the listener